contracts/masset/platform-integrations/CompoundIntegration.sol

```diff
@@ -16,6 +16,8 @@ contract CompoundIntegration is InitializableAbstractIntegration {
     event RewardTokenCollected(address recipient, uint256 amount);
     event SkippedWithdrawal(address bAsset, uint256 amount);
 
+    uint256 internal lastClaimed;
```

Comment:
Does this work with the upgradeable pattern?

Reply:
Yes, it will be initialised with the value 0, so should execute on the first deposit/withdrawal